src/routes/events.$eventId.kiosk.tsx

```
@@ -86,14 +129,71 @@ export default function KioskMode() {
+          password: Math.random().toString(36).substring(2, 10).toUpperCase(),
+          essid: `${hostCollege}-Guest`,
+          expires_at: new Date(Date.now() + 12 * 60 * 60 * 1000).toISOString(),
+        };
+
+        try {
+          // Trigger secure webhook to Cisco ISE mock
+          const response = await fetch("https://api.cisco-ise.local/v1/guest-provision", {
+            method: "POST",
+            headers: {
+              "Content-Type": "application/json",
+              "Authorization": "Bearer cisco-ise-secret-token",
```

ESLint

[error] Replace `"Authorization"` with `Authorization`

(prettier/prettier)